user/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from .models import User
```

Comment:
ローカルなアプリケーションのimportは1行追加で開けるとさらにgoodです！
https://pep8-ja.readthedocs.io/ja/latest/#import も参考にしてみてください。

autopep8に加えて，isortというライブラリを用いるとこちらも自動化できるので試してみてください！
```suggestion
from django.contrib import admin
from django.contrib.auth.admin import UserAdmin

from .models import User
```